app/assets/javascripts/logs.coffee

```diff
@@ -70,6 +72,19 @@ class Log
 
     updateTotalHours()
 
+    projectChangeListener = ->
+      $("select[id$='_project_id']").on "change", distinctProjects
+      $("select[id$='_project_id']").each distinctProjects # Doesn't work yet.
+
+    distinctProjects = ->
+      console.log("hit it")
+      $("select[id$='_project_id']").children().removeAttr 'disabled'
```

Comment:
`$("select[id$='_project_id']")` is called multiple times and can be refactored into a variable